lib/hummingbird/impl.ex

```diff
@@ -11,7 +11,8 @@ defmodule Hummingbird.Impl do
     %{
       conn
       | private: nil,
-        secret_key_base: nil
+        secret_key_base: nil,
+        assigns: Map.take(conn.assigns, [:current_user, :admin_user])
```

Comment:
```suggestion
        assigns: nil,
        selected_assigns: Map.take(conn.assigns, [:current_user, :admin_user])
```

Reply:
Will let us differentiate the actual assigns should we throw it back in, from what we're doing today.

I think the appropriate default behaviour in the longer term is to allow plug-user to define an allow-list of patterns?

Feel free to not accept change and we can discuss that way of working later.